modules/users/client/services/password-validator.client.service.js

```diff
@@ -11,7 +11,7 @@ angular.module('users').factory('PasswordValidator', ['$window',
         return result;
       },
       getPopoverMsg: function () {
-        var popoverMsg = 'Please enter a passphrase or password with greater than 10 characters, numbers, lowercase, upppercase, and special characters.';
+        var popoverMsg = 'Please enter a passphrase or password with more than 10 characters, numbers, lowercase, uppercase, and special characters.';
```

Comment:
@ilanbiala I think the point is to give the User a heads up on what the requirements are for the password. Rather than displaying the message on the page, I think this is a good way to convey this info to the User.